### PR TITLE
build: upload arm64 macOS files with different filenames

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

**Comment by jazzdan** [comment]
This is necessary for my solution https://github.com/replayio/backend/pulls?q=is:pr+is:open+sort:updated-desc to work

File: replay_build_scripts/upload_build_artifacts.mjs [hunk 1]
```diff
@@ -155,9 +157,9 @@ function prepareWindowsBinaries(buildId) {
   return [buildArchive];
 }
 
-function prepareMacOSBinaries(buildId) {
-  const dmgArchive = `${buildId}.dmg`;
-  const outdir = path.join("out", "Release");
+function prepareMacOSBinaries(buildId, buildArm) {
```
Comment (ryanjduffy) [inline replay_build_scripts/upload_build_artifacts.mjs:160]: Adding `buildArm` as a argument here will shadow the module variable causing it to always be `undefined`, right?

Comment (jazzdan) [reply replay_build_scripts/upload_build_artifacts.mjs:160]: Ugh. I miss TypeScript!

Comment (jazzdan) [reply replay_build_scripts/upload_build_artifacts.mjs:160]: (Thanks!)

**Pull request merged**